datastore/kafka/kafka.go

```diff
@@ -74,21 +72,17 @@ func (p *Producer) Produce(entry *telemetry.Record) {
 		Key:            []byte(entry.Vin),
 		Headers:        headersFromRecord(entry),
 		Timestamp:      time.Now(),
+		Opaque:         entry,
 	}
 
 	// Note: confluent kafka supports the concept of one channel per connection, so we could add those here and get rid of reliableAckWorkers
 	// ex.: https://github.com/confluentinc/confluent-kafka-go/blob/master/examples/producer_custom_channel_example/producer_custom_channel_example.go#L79
```

Comment:
I wonder if there's a better place for this comment now?

Reply:
hmm I would prefer to keep it at this location since we will update how `p.kafkaProducer.Produce(msg, nil)` works.  eventually, it will be powered through eventDeliveryChannel